### DISCUSSION TITLE
How to enable the rx/tx activity chart/graph

### DISCUSSION THREAD

**Discussion by sym0nd0** [wg-easy/wg-easy#908] [Q&A] [answered]
Hi,
I'm trying to get the activity visual to show like [those on the readme.md](https://github.com/wg-easy/wg-easy/blob/master/assets/screenshot.png).

Mine just won't show it and I can't find anyway to enable it or any results from searchs for anyone else that has had the same issue. 😕
![image](https://github.com/wg-easy/wg-easy/assets/55590428/fbacf686-5cbd-4bcb-b678-ae6db49b15c3)

I have just enabled 'UI_TRAFFIC_STATS' but with or without that doesn't change in that respect, it does give some sweet extra stats though.

Running on latest, as of an hour ago but previous version was the same.

Thanks!

**Comment by pheiduck** [comment, marked as answer]
Will be fixed with: https://github.com/wg-easy/wg-easy/pull/911

**Reply by sym0nd0** [reply]
Amazing, thanks. No clue how I didn't find those issues.
